ui/eureka/eureka.js

```diff
@@ -1817,6 +2048,13 @@ class EurekaTracker {
 
   OnZoneChange(e) {
     this.zoneName = e.detail.zoneName.replace('The Forbidden Land, ', '');
+    this.zoneName = this.zoneName.replace('禁地优雷卡 ', '');
+    let zones = this.options.ZoneName[this.options.Language] || this.options.ZoneName['en'];
+    for (let zone in zones) {
+      this.zoneName = this.zoneName.replace(
+          this.options.ZoneName[this.options.Language][zone],
```

Comment:
Shouldn't this line be `zones[zone],`? It's much shorter and doesn't throw an error (like #687) if `this.options.Language` doesn't exist in the `ZoneName` object.

Reply:
Yes, should be zones[zone] here, otherwise languages except for en and cn won’t work here, sorry.

Reply:
fixed in https://github.com/quisquous/cactbot/pull/688